src/main/java/com/brandwatch/internship/fetchandstoretweets/entities/Mention.java

```diff
@@ -0,0 +1,93 @@
+package com.brandwatch.internship.fetchandstoretweets.entities;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.Date;
+
+public class Mention {
+
+    private static final long serialVersionUID = 1L;
+    private final long id;
```

Comment:
I think your ids can just be `int` (queryId as well). Any specific reason why you chose `long`?

Reply:
I use the same ids of the tweets and they are long as I can remember.

Reply:
Ah I see. If you plan to do query matching at some point, then you should give mentions your own ids as one tweet potentially can create multiple mentions.

Reply:
I can make mentions ids and tweets's ids different but I think this will cause duplicate mentions. Actually if we have a mention created at time t, then at t + 20s this mention will be duplicated. Right now it just overwrites it by the saveall() method.